public/static/docs/command-reference/gc.md

```diff
@@ -14,9 +15,13 @@ usage: dvc gc [-h] [-q | -v] [-a] [-T] [-c] [-r REMOTE] [-f] [-j JOBS]
 This command deletes (garbage collects) data files or directories that may exist
 in the cache (or [remote storage](/doc/command-reference/remote) if `-c` is
 used) but no longer referenced in [DVC-files](/doc/user-guide/dvc-file-format)
-currently in the <abbr>workspace</abbr>. By default, this command only cleans up
-the local cache, which is typically located on the same machine as the project
-in question. This usually helps to free up disk space.
+currently in the <abbr>workspace</abbr>. By default, this command won't delete
+anything at all to make it safe and explicit. However, you can use different
+flags to change the behavior.
```

Comment:
Maybe `anything at all (a warning is shown),` but yeah, maybe that's not relevant, up to you.

Reply:
I checked on DVC 0.87.0 and actually running without flags causes an ERROR output (`ERROR: unexpected error - Invalid Arguments. Either of (all_branches, all_tags, all_commits, workspace, cloud) needs to be enabled.`). Maybe we should express this differently like:
```suggestion
currently in the <abbr>workspace</abbr>. To avoid accidentally deleting data, this command
requires the explicit use of explicit [options](#options) to determine it's behavior (what
"garbage" to collect).
```
> Prob needs wraping.